target/check-for-changes.sh

```diff
@@ -1,11 +1,15 @@
-#! /bin/bash
+#!/bin/bash
 
 # create date for log output
 log_date=$(date +"%Y-%m-%d %H:%M:%S ")
 # Prevent a start too early
 sleep 5
 echo "${log_date} Start check-for-changes script."
 
+# create work area outside mounted directory
+mkdir -p /tmp/docker-mailserver-work
```

Comment:
could it be an idea to work with mktmp here to prevent accidentally using leftovers from other runs?

Reply:
In this case no, the directory can be reused by the **same container** without ill effects (in fact it should as a change in checksums should trigger updates of the files in /etc). The problem I'm trying to fix is where multiple containers share the same checksum file in the mounted directory. We could put it in /var/something instead if that makes it clearer.

Reply:
hm.. have not looked that deep into this part of this project yet. So if I see this correctly then this script just writes a file that then includes the different checksums. In that case I would not even create a subdir for this in /tmp but rather give the file a descriptive name and place it directly into /tmp.

since the information in this file can be easily regenerated I would not put this into /var

Reply:
Well, I dislike clutter and thought that there could be other similar files later that could live in the same place, but we can certainly remove the directory and put the file directly in /tmp. We can always add a directory later.